docs/api/qiskit-ibm-provider/_toc.json

```diff
@@ -6,7 +6,7 @@
       "children": [
         {
           "title": "Overview",
-          "url": "/api/qiskit-ibm-provider/ibm_provider"
+          "url": "/api/qiskit-ibm-provider/index"
```

Comment:
Should we maybe link to just `/api/qiskit-ibm-provider/` (i.e. without `/index`)?

This is a bikeshed comment though so just go with whatever you prefer.

Reply:
Eh, couldn't figure out quickly how to do this so I'm going to punt for now: https://github.com/Qiskit/documentation/issues/361. It's behaving this way for Qiskit too